Resources/Prototypes/_RMC14/Entities/Objects/Medical/surgical_tray.yml

```diff
@@ -0,0 +1,38 @@
+- type: entity
+  parent: CMSurgicalCase
+  id: CMSurgicalTray
```

Comment:
Should be RMC for new content and not CM
make sure you update it for the other usages in this pr as well